### DISCUSSION TITLE
Feedback: New issue templates!

### DISCUSSION THREAD

**Discussion by phated** [gulpjs/gulp#2443] [Ideas]
Hey all! Now that we have Discussions set up, we wanted to revamp the `ISSUE_TEMPLATES` to help organize conversations into the appropriate channels.

We did a first pass at them today, but we already thinking about improvements to be mad like creating templates for documentation issues.

We're interested other feedback you might have or other templates we could add!

**Comment by Miladiir** [comment]
Am I allowed to post constructive criticism about the contents of the documents, or will I receive a temporary ban for that as well?

**Reply by phated** [reply]
This sure feels sarcastic, but I'll entertain it. We are continuing to iterate our issue templates to better explain to people that might be new to open source better ways to contribute in certain scenarios.

As for bans: our general rule is "don't make it miserable for the contributors to work on this project or you will be banned"
